g2p-transform.cwl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env cwl-runner
+
+# this should produce a docker run of the form:
+# docker run --rm -it \
+# -e "MOLECULAR_MATCH_API_KEY=XXXXXXXX" \
+# -v /tmp/g2p:/output \
+# -v ~/.synapseCache/.session:/root/.synapseCache/.session  \
+# -v /tmp/g2p/cgi_biomarkers_per_variant.tsv:/g2p-aggregator/harvester/cgi_biomarkers_per_variant.tsv \
+# -v /tmp/g2p/cosmic_lookup_table.tsv:/g2p-aggregator/harvester/cosmic_lookup_table.tsv  \
+# -v /tmp/g2p/harvester.sqlite:/g2p-aggregator/harvester/harvester.sqlite \
+# -v /tmp/g2p/oncokb_all_actionable_variants_20170822.tsv:/g2p-aggregator/harvester/oncokb_all_actionable_variants.tsv \
+# g2p-transform
+
+# note: the volumes & swiftPath attribute is somewhat ad-hoc, there is no official support for openstack swift
```

Comment:
I think at this level, it should be more of an object name, the fact that swift or s3 is behind the scenes shouldn't matter